cogs/StaffAppsBackoffice.py

```diff
@@ -0,0 +1,966 @@
+            data = self.cursor.execute(
+                "select appId, discordName, description, submissionTime from application a join status s on a.status = s.statusId order by appId ASC",
+            ).fetchall()
+            title = f"Applications page 1/{len(data)//10+1}"
+            field_title = "Application ID, Discord Name, Status, Submission Time"
+            field_content = ""
+            for row in data[:10]:
+                field_content += f"{row[0]}, {row[1]}, {row[2]}, {datetime.fromtimestamp(int(row[3]))}\n"
+            fields = [[field_title, field_content, False]]
+            embed = EmbedBuilder(
+                title=title,
+                description="",
+                fields=fields,
+                color=0x24AAE1,
+            )  # hwh blue
+            if len(data) > 10:
+                await ctx.respond(
+                    embed=embed.build(),
+                    view=staffAppsSeeAll(ctx.author, data),
+                    ephemeral=True,
+                )
+            else:
+                await ctx.respond(embed=embed.build(), ephemeral=True)
+            Log(f"{username} viewed all applications")
+            return
+
+        elif subcommand == "denied":
+            data = self.cursor.execute(
+                "select appId, discordName, submissionTime from application where status = 2",
+            ).fetchall()
+            title = f"Denied Applications page 1/{len(data)//10+1}"
+            field_title = "Application ID, Discord Name, Original Submission Time"
+            field_content = ""
+            for row in data[:10]:
+                field_content += (
+                    f"{row[0]}, {row[1]}, {datetime.fromtimestamp(int(row[2]))}\n"
+                )
+            fields = [[field_title, field_content, False]]
+            embed = EmbedBuilder(
+                title=title,
+                description="",
+                fields=fields,
+                color=0x9D0006,
+            )  # soft red
+            if len(data) > 10:
+                await ctx.respond(
+                    embed=embed.build(),
+                    view=staffAppsSeeDeniedAccepted(ctx.author, data, "d"),
+                    ephemeral=True,
+                )
+            else:
+                await ctx.respond(embed=embed.build(), ephemeral=True)
+            Log(f"{username} viewed denied applications")
+            return
+
+        elif subcommand == "accepted":
+            data = self.cursor.execute(
+                "select appId, discordName, submissionTime from application where status = 9",
+            ).fetchall()
+            title = f"Accepted Applications page 1/{len(data)//10+1}"
+            field_title = "Application ID, Discord Name, Original Submission Time"
+            field_content = ""
+            for row in data[:10]:
+                field_content += (
+                    f"{row[0]}, {row[1]}, {datetime.fromtimestamp(int(row[2]))}\n"
```

Comment:
datetime.fromtimestamp() takes an EPOCH timestamp as argument, EPOCH timestamps are the time in seconds elapsed since 00:00:00 UTC on 1 January 1970. This object is already timezone-aware at the time of creation.